Sources/MintKit/Mint.swift

```diff
@@ -324,6 +326,32 @@ public class Mint {
         }
     }
 
+    private func installManPages(packagePath: PackagePath) throws {
+        let package = packagePath.package
+        let checkoutPath = Path.temporary + "mint"
+        let packageCheckoutPath = checkoutPath + packagePath.repoPath
```

Comment:
These are duplicated lines of code from caller of this function.
Should I refactor to include these into `PackagePath` ?